### ISSUE TITLE
Debug console logs black text on dark themes

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#76550]
Launch VS Code debug target:

![image](https://user-images.githubusercontent.com/2193314/60629787-06786500-9dac-11e9-9e30-a90a01fef018.png)

FYI terminal apps work around this sort of problem by inverting the color instead of printing white and black.

@lramos15 spotted this earlier today

**Comment by isidorn** [comment]
What debug extension are you debugging @lramos15 @Tyriar 
I can not reproduce this with regular console output in abyss theme. It seems like it is some special ansi color which gets printed in your case which looks wrong.

![Screenshot 2019-07-05 at 16 52 03](https://user-images.githubusercontent.com/1926584/60730366-3b6bef80-9f45-11e9-939c-68275b33df89.png)

**Comment by lramos15** [comment]
Chrome debug

**Comment by Tyriar** [comment]
@roblourens maybe this is your domain? Prefer `\x1b[7m` to invert text over using black or white.

**Comment by roblourens** [comment]
We are getting this, and doing as told: `{"method":"Runtime.consoleAPICalled","params":{"type":"info","args":[{"type":"string","value":"%c[Extension Host] %cdebugger listening on port 5870"},{"type":"string","value":"color: blue"},{"type":"string","value":"color: black"}],`

Not sure how to handle that...

**Comment by Tyriar** [comment]
It's here:

https://github.com/microsoft/vscode/blob/7e03eb1a1fce3beb6ccea5f3b8f828e01940be61/src/vs/workbench/services/extensions/electron-browser/extensionHost.ts#L293

Can this be `color: inverse` instead, and add support to debug console if it doesn't support that yet?

**Comment by roblourens** [comment]
I think we just want to reset the color there, and this is what works in chrome devtools:

![image](https://user-images.githubusercontent.com/323878/60740751-b4565180-9f1b-11e9-9dae-164f55fe9d1c.png)

But not in vscode so I just just fix that
